selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -267,14 +268,12 @@ void AdvancedNetworking::toggleTethering(int enable) {
 }
 
 void enableSSH(Toggle* toggle_switch_SSH){
-  system("sudo systemctl enable ssh");
-  system("sudo systemctl start ssh");
+  Params().write_db_value("SshEnabled", "1");
```

Comment:
Shouldn't we keep the `systemctl start`? Do we really need a param for this? Now there is state about SSH in two places (the param and the systemd symlink) which can get out of sync.

Reply:
the symlink doesn’t work on a ro rootfs, so we have a systemd service that triggers on the ssh param changing

Reply:
If there the param is the only source of truth for ssh being on it's fine!